### PR TITLE
Refactor catalog plays for site playbook

### PR DESCRIPTION
**Pull request by adborden**
- Splits catalog-web and catalog-harvester plays
- Removes unused roles

### CODE REVIEW

**Comment by adborden** [comment]
This replaces the catalog work in https://github.com/GSA/datagov-deploy/pull/610

File: ansible/site.yml [hunk 1]
```diff
@@ -10,3 +10,8 @@
 - import_playbook: nessus.yml
 - import_playbook: newrelic-infrastructure.yml
 - import_playbook: unattended-upgrades.yml
+
+
+# App plays
+############
+- import_playbook: catalog.yml
```
Comment (JJediny) [inline ansible/site.yml:17]: Was this meant to be included? I'm deleting the site.yml playbook altogether with the refactoring work

**Review by JJediny — COMMENTED** [review]
Looks good but can you change the ckan-worker to ckan-harvest as there is another kind of ckan instance that is the qa worker and I don't want to be mixing vocabs

**Review by JJediny — APPROVED** [review]
LGTM

**Pull request merged**